docs/1-introduction/1-1-getting-started/1-1-2-hello-d3d11.md

```diff
@@ -185,15 +185,15 @@ The first part calls the parent class, where `GLFW` is initialized and setup.
 ```cpp
 #define IID_PPV_ARGS(ppType) __uuidof(**(ppType)), IID_PPV_ARGS_Helper(ppType)
 ```
-which means that typing `IID_PPV_ARGS(&_dxgiFactory)` it is expanded by the compiler into `__uuidof(**(&_dxgiFactory)), IID_PPV_ARGS_Helper(_dxgiFactory)`. This funtionally means that for functions that have a parameter setup as `REFIID` and immediatly after a `[out] void**` parameter, this macro will expand the `IID_PPV_ARGS(ppType)` expression into these parameters for ease of the programmer — this can be seen with the used `CreateDXGIFactory2` method where the second last and last parameter are a `REFIID` and `void**`:
+which means that typing `IID_PPV_ARGS(&_dxgiFactory)` it is expanded by the compiler into `__uuidof(**(&_dxgiFactory)), IID_PPV_ARGS_Helper(_dxgiFactory)`. This funtionally means that for functions that have a parameter setup as `REFIID` and immediately after a `[out] void**` parameter, this macro will expand the `IID_PPV_ARGS(ppType)` expression into these parameters for ease of the programmer — this can be seen with the used `CreateDXGIFactory2` method where the second last and last parameter are a `REFIID` and `void**`:
```

Comment:
`for ease of the programmer` sounds funny too. 

`for ease of use` might be a better way of saying it?